packages/daemon/src/daemon.js

```diff
@@ -39,33 +44,149 @@ const sinkError = error => {
 
 /**
  * @param {import('../index.js').Locator} locator
+ * @param {string} sha512
  */
-const makeWorker = async locator => {
-  // @ts-ignore Node.js crypto does in fact have randomUUID.
-  const uuid = await crypto.randomUUID();
-  const workerCachePath = path.join(locator.cachePath, uuid);
-  await fs.promises.mkdir(workerCachePath, { recursive: true });
-  const logPath = path.join(workerCachePath, 'worker.log');
-  const output = fs.openSync(logPath, 'w');
-  const child = popen.fork(endoWorkerPath, [uuid, workerCachePath], {
-    stdio: ['ignore', output, output, 'pipe', 'ipc'],
+const provideReadableSha512 = (locator, sha512) => {
+  return Far(`Readable file with SHA-512 ${sha512.slice(0, 8)}...`, {
+    sha512() {
+      return sha512;
+    },
+    stream() {
```

Comment:
I added a `text()` method a la:

```diff
index c8b964637..42e6bbd71 100644
--- a/packages/daemon/src/daemon.js
+++ b/packages/daemon/src/daemon.js
@@ -47,13 +47,21 @@ const sinkError = error => {
  * @param {string} sha512
  */
 const provideReadableSha512 = (locator, sha512) => {
+  const getPath = () => {
+    const storageDirectoryPath = path.join(locator.statePath, 'store-sha512');
+    const storagePath = path.join(storageDirectoryPath, sha512);
+    return storagePath;
+  };
+
   return Far(`Readable file with SHA-512 ${sha512.slice(0, 8)}...`, {
     sha512() {
       return sha512;
     },
+    text() {
+      return fs.promises.readFile(getPath(), 'utf8');
+    },
     stream() {
-      const storageDirectoryPath = path.join(locator.statePath, 'store-sha512');
-      const storagePath = path.join(storageDirectoryPath, sha512);
+      const storagePath = getPath();
       const nodeReadStream = fs.createReadStream(storagePath);
       const reader = makeNodeReader(nodeReadStream);
       return makeReaderRef(reader);
~
```

Reply:
that allowed me to do:

```
$ ./bin/endo eval w1 'E(mkc).text()' mkc
const makeCounter = () => {
  let value = 0;
  return {
    incr: () => (value += 1),
    decr: () => (value -= 1),
  };
};

makeCounter;
```

but this hangs:

```
./bin/endo eval w1 'E(mkc).text().then(eval)' mkc
```

I couldn't find a worker log. (I did find a worker .pid file.)